assets/src/ba_data/python/ba/_gameutils.py

```diff
@@ -46,6 +47,14 @@ def get_trophy_string(trophy_id: str) -> str:
         return _ba.charstr(TROPHY_CHARS[trophy_id])
     return '?'
 
+def distance(vec1: Tuple[float, float, float],
+             vec2: Tuple[float, float, float]) -> float:
+    """Find distance between two positions"""
+    xlen = vec1[0] - vec2[0]
+    ylen = vec1[1] - vec2[1]
+    zlen = vec1[2] - vec2[2]
+    xylen = math.sqrt(xlen**2 + ylen**2)
+    return math.sqrt(xylen**2 + zlen**2)
```

Comment:
Same note as down below; I'd prefer to go through ba.Vec3 as much as possible for functionality like this instead of doing raw number crunching in Python. Over time I will have more node attributes/etc. return ba.Vec3 instead of lists/tuples to hopefully make vector math as easy as possible. If there's anything coming out incorrect with the ba.Vec3 methods or anything that could be easier to use, please let me know.